x-pack/plugins/observability_ai_assistant/server/service/client/index.ts

```diff
@@ -406,7 +412,7 @@ export class ObservabilityAIAssistantClient {
           subscriber.complete();
         };
 
-        next(messages).catch((error) => {
+        next(this.addResponseLanguage(messages, responseLanguage)).catch((error) => {
```

Comment:
It would be great  if we had some simple unit tests that displayed the generated messages based on an initial user prompt. For instance, adding the language request to the prompt would be easier to understand if a unit test showed it.

Reply:
Sure, that's easy to add!